package.json

```diff
@@ -27,6 +26,7 @@
     "fix:js": "npm run test:eslint:js -- --fix",
     "fix:scss": "npm run test:scss -- --fix",
     "heroku-postbuild": "npm run build",
+    "optimize:css": "postcss network-api/networkapi/frontend/_css/*.css --dir network-api/networkapi/frontend/_css",
```

Comment:
this covers all CSS files in that dir

Reply:
We probably want a better name for this task as our `postcss` setup doesn't only cover optimization (minification) but also autofixer. Suggestions